File: commands/meMessage.js
'use strict';
function meMessage(msg, args) {
	msg.delete();
	if (args[0] === 'анон' && args.length > 1) {
		args.shift();
		const textString = args.join(' ');
		msg.channel.send(textString);
		return;
	}
	else if (args[0] === 'анонттс' && args.length > 1) {
		args.shift();
		const textString = args.join(' ');
		msg.channel.send(textString, { tts: true });
		return;
	}
	else if (args[0] !== 'анон' && args.length > 0) {
		const textString = args.join(' ');
		msg.channel.send(`${msg.author} ${textString}`);
		return;
	}
}

module.exports = {
	name: 'meMessage',
	description: 'Module reproduces famous command /me',
	cooldown: 2,
	execute(msg, args) {
		meMessage(msg, args);
	},
};

File: src/commands/meMessage.py
async def meMessage(msg, args):
    await msg.delete()
    if len(args) >= 1:
        if args[0] == 'анон':
            args.pop(0)
            textString = " ".join(args)
            await msg.channel.send(textString)
        elif args[0] == 'анонттс':
            args.pop(0)
            textString = " ".join(args)
            await msg.channel.send(textString, tts=True)
        else:
            textString = " ".join(args)
            await msg.channel.send(f'{msg.author.mention} {textString}')
    else:
        return
